src/snmalloc/mem/corealloc.h

```diff
@@ -401,7 +405,7 @@ namespace snmalloc
         meta->node.remove();
 
         Config::Backend::dealloc_chunk(
-          get_backend_local_state(), *meta, p, size);
+          get_backend_local_state(), *meta, p, size, 0);
```

Comment:
Large objects always have zero metadata space?

Reply:
There is always one in the metaslab by default.  But I think that might be giving me UB.

Reply:
Made this clearer, and added a comment when I refactored where the extra_bytes are calculated.